services/web/src/screens/Users/List.js

```diff
@@ -0,0 +1,167 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { formatDateTime } from 'utils/date';
+import { request } from 'utils/api';
+import { Layout } from 'components/Layout';
+import { screen } from 'helpers';
+
+import { Confirm } from 'components/Semantic';
+import SearchProvider from 'components/SearchProvider';
+import HelpTip from 'components/HelpTip';
+import { EditUser, Filters } from 'modals';
```

Comment:
As per our discussion: In the interest of reducing friction on this PR and number of steps to create a component we decided to forgo the index files for now.

Reply:
ok removed `screens/index.js` and `modals/index.js` ... since the generator no longer has to deal with these I think it's over all a good thing

I would like to make one last case to keep `components/index.js` before refactoring as components are more common to mix and match a lot together in the same way you import semantic... for example this:

```js
import { Layout, Confirm, HelpTip, SearchProvider } from 'components';
```

would have to be this:

```js
import Layout from 'components/Layout';
import Confirm from 'components/Confirm';
import HelpTip from 'components/HelpTip';
import SearchProvider from 'components/SearchProvider';
```

if we really want to go back to this however I'll make the change

Reply:
@andrewplummer Not sure why this needs to happen in this particular branch, but sure go ahead. Just make sure all components are in that index?

Reply:
it doesn't it's just a lot to roll back now that I'd have to do again if I'm able to convince you so wanted to give it a shot :)

Reply:
Yep fine